changelog.d/1580

```diff
@@ -0,0 +1,10 @@
+synopsis: Allow IO in validationKeys
+prs: #1580
+issues: #1579
+
+description: {
+
+Currently validationKeys are a fixed JWKSet. This does not work for setups such
```

Comment:
Can we generalise this to:

This does not work with OIDC providers like for example Okta and AWS Cognito, which require regularly fetching the `jwks_uri` to discover expired and new keys. 

(As it's not Okta specific; it's OIDC specific)

Reply:
Done!